core/state_transition.go

```diff
@@ -333,6 +329,7 @@ func (st *StateTransition) preCheck() error {
 	if st.evm.ChainConfig().IsCancun(st.evm.Context.BlockNumber, st.evm.Context.Time) {
 		if st.blobGasUsed() > 0 {
 			// Check that the user is paying at least the current blob fee
+			// ExcessBlobGas must not be nil, which is checked in eip4844.VerifyEIP4844Header
```

Comment:
No need for this log. The code would get very very full of these if we were to add all kinds of notes on why one field or another cannot be nil.